esi_leap/common/flocx_market.py

```diff
@@ -0,0 +1,54 @@
+from esi_leap.db import api as dbapi
+from esi_leap.objects import flocx_market_client
+from esi_leap.objects import offer
+from keystoneauth1 import adapter
+from keystoneauth1 import loading as ks_loading
+from oslo_config import cfg
+import datetime
+
+CONF = cfg.CONF
+
+def get_flocx_market_client():
+    auth_plugin = ks_loading.load_auth_from_conf_options(
+        CONF, 'flocx_market')
+    sess = ks_loading.load_session_from_conf_options(CONF, 'flocx_market',
+                                                         auth=auth_plugin)
+    adpt = adapter.Adapter(
+        session=sess,
+        service_type='marketplace',
+        interface='public')
```

Comment:
We shouldn't be hardcoding the `interface` here, but that can be fixed in a future PR.